web/testing/caps-lock-layer-3620/caps_lock_layer_3620.js

```diff
@@ -0,0 +1,1661 @@
+if(typeof keyman === 'undefined') {
+  console.log('Keyboard requires KeymanWeb 10.0 or later');
+  if(typeof tavultesoft !== 'undefined') tavultesoft.keymanweb.util.alert("This keyboard requires KeymanWeb 10.0 or later");
+} else {
+KeymanWeb.KR(new Keyboard_caps_lock_layer_3620());
+}
+function Keyboard_caps_lock_layer_3620()
+{
+  var modCodes = keyman.osk.modifierCodes;
+  var keyCodes = keyman.osk.keyCodes;
+
+  this._v=(typeof keyman!="undefined"&&typeof keyman.version=="string")?parseInt(keyman.version,10):9;
+  this.KI="Keyboard_caps_lock_layer_3620";
+  this.KN="Caps Lock Layer 3620";
+  this.KMINVER="10.0";
+  this.KV={F:' 1em "Arial"',K102:0};
+  this.KV.KLS={
+
+  };
```

Comment:
```suggestion
  this.KV.KLS={ };
```

Reply:
This is compiled code; only reason I've included it in the PR is to make it available for testers.